### ISSUE TITLE
Question about loss

### ISSUE THREAD

**Issue by WeihuangLin** [Code-kunkun/LamRA#12]
Hi @Code-kunkun,

In your code, https://github.com/Code-kunkun/LamRA/blob/72863991e62f364d734231b6e3327a985c229984/models/qwen2_vl.py#L190 you've only designed a contrastive learning loss between embeddings.  Why didn't you design a loss between labels and logits?

**Comment by Code-kunkun** [comment]
Since our goal is for the large multimodal model to perform retrieval tasks rather than generation tasks, our loss consists only of contrastive learning between embeddings.
